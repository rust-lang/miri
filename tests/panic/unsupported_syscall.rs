//@ignore-target-windows: No libc on Windows
//@ignore-target-apple: `syscall` is not supported on macOS
//@compile-flags: -Zmiri-panic-on-unsupported

fn main() {
    unsafe {
        libc::syscall(0);
    }
}
