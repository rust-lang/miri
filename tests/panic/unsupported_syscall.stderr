thread 'main' panicked at 'unsupported Miri functionality: can't execute syscall with ID 0', $DIR/unsupported_syscall.rs:LL:CC
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
